src/util/directive-controller.js

```diff
@@ -27,7 +25,7 @@ export default function createDirectiveController(caller, injects, controller, d
   // Remember, angular has alrady set those bindings on the prototype of the calling
   // function. Now we need to extend them onto our instance. important
   // to extend after building the properties that way we fire the setters
-  extend(instance, caller);
+  Object.assign(instance, caller);
```

Comment:
Internet Explorer doesn't have `Object.assign`, so we can't use that here, sorry!